Dockerfile.image

```diff
@@ -10,12 +10,23 @@ COPY framework/files/etc/luet/luet.yaml /etc/luet/luet.yaml
 ENV LUET_NOLOCK=true
 SHELL ["/usr/bin/luet", "install", "-y", "--system-target", "/framework"]
 
+# elemental-toolkit essentials
+RUN system/immutable-rootfs
```

Comment:
cant we just use the `meta/cos-light` which was the package used to sync the framework files before?

Reply:
probably, I have to admit I have not checked the meta-packages. Gonna do it :+1: